ZenKeySDK/Sources/Core/DiscoveryService.swift

```diff
@@ -55,6 +55,27 @@ protocol DiscoveryServiceProtocol {
     func discoverConfig(forSIMInfo simInfo: SIMInfo?,
                         prompt: Bool,
                         completion: @escaping DiscoveryServiceCompletion)
+
+    /// Fetch the cached config that maps to sim.
+    /// - Parameters:
+    ///   - sim: The sim info to pass to the discovery service
+    /// - Returns: Cached instance that conforms to ScopeZen.
+    func cachedScopezen(at sim: SIMProtocol) -> ScopeZen?
```

Comment:
nit: should cachedScopezen have a capital Z?

Reply:
yes